tools/xcodeproj_shims/installers/_swiftmodule.sh

```diff
@@ -5,15 +5,24 @@ set -euo pipefail
 for module in "$@"; do
     doc="${module%.swiftmodule}.swiftdoc"
     module_name=$(basename "$module")
-    module_bundle="$BUILT_PRODUCTS_DIR/$module_name"
-    mkdir -p "$module_bundle"
-
-    cp "$module" "$module_bundle/$NATIVE_ARCH_ACTUAL.swiftmodule"
-    cp "$doc" "$module_bundle/$NATIVE_ARCH_ACTUAL.swiftdoc"
+	
+    dest_products_module_bundle="$BUILT_PRODUCTS_DIR/$module_name"
+	
+    mkdir -p "$dest_products_module_bundle"
 
+    cp "$module" "$dest_products_module_bundle/$NATIVE_ARCH_ACTUAL.swiftmodule"
+    cp "$doc" "$dest_products_module_bundle/$NATIVE_ARCH_ACTUAL.swiftdoc"
+	
     ios_module_name="$NATIVE_ARCH_ACTUAL-$LLVM_TARGET_TRIPLE_VENDOR-$SWIFT_PLATFORM_TARGET_PREFIX$LLVM_TARGET_TRIPLE_SUFFIX"
-    cp "$module" "$module_bundle/$ios_module_name.swiftmodule"
-    cp "$doc" "$module_bundle/$ios_module_name.swiftdoc"
+    cp "$module" "$dest_products_module_bundle/$ios_module_name.swiftmodule"
+    cp "$doc" "$dest_products_module_bundle/$ios_module_name.swiftdoc"
 
-    chmod -R +w "$module_bundle"
+    chmod -R +w "$dest_products_module_bundle"
+	
+	# XCode expects to find swiftmodules in Build/Intermediates and has a build phase where it copies
+	# the Intermediates swiftmodules to Built Products Dir using `ditto`
```

Comment:
This is an unfortunate additional step. Even if the xcode build setting DEFINES_MODULE is false, XCode will still try to copy the swiftmodule from Intermediates into Build/Products dir using `ditto`. If the intermediates file doesn't exist, it will generate an empty swiftmodule in Build/Products!